config.js

```diff
@@ -1,11 +1,11 @@
 window.config = {
 
 	// the url of the DonorDrive API you wish to poll
-	api: 'https://docs.donordrive.com/api/',
+	api: 'https://www.extra-life.org/api/',
```

Comment:
this should still point to our "demo" site - while Extra Life constitutes a large percentage of our streamers, we have fundraisers for other orgs as well (and this participant wont go away at the end of the year)

Reply:
You are absolutely right, accidentally left that in when testing based on my current fundraiser.